src/test/java/spoon/test/variable/VariableTest.java

```diff
@@ -191,4 +200,27 @@ public void testVarInLambda() {
         assertEquals("java.lang.Long", lambda.getParameters().get(1).getType().getQualifiedName());
         assertEquals("(var x,var y) -> x + y", lambda.toString()); // we should print var, if it was in the original code
     }
+
+    @ModelTest(value = "./src/test/resources/spoon/test/unnamed/UnnamedVar.java", complianceLevel = 22)
+    void testUnnamedVariable(Factory factory) throws IOException {
```

Comment:
Contract? I have no idea what this test tests :P That the lines match the source and everything is unnamed?